src/SFA.DAS.ApprenticeCommitments/Data/Models/Revision.cs

```diff
@@ -95,16 +95,17 @@ private void ConfirmRevision(DateTimeOffset time)
             bool EmployerIsEquivalent() => details.EmployerIsEquivalent(Details);
             bool ProviderIsEquivalent() => details.ProviderIsEquivalent(Details);
             bool ApprenticeshipIsEquivalent() => details.ApprenticeshipIsEquivalent(Details);
+            bool DeliveryModelIsEquivalent() => Details.DeliveryModel == details.DeliveryModel;
 
             if (Details.Equals(details)) return null;
 
             var revision = new Revision(commitmentsApprenticeshipId, approvedOn, details);
 
-            if (EmployerIsEquivalent()) revision.EmployerCorrect = EmployerCorrect;
+            if (EmployerIsEquivalent() && DeliveryModelIsEquivalent())  revision.EmployerCorrect = EmployerCorrect;
             if (ProviderIsEquivalent()) revision.TrainingProviderCorrect = TrainingProviderCorrect;
             if (ApprenticeshipIsEquivalent()) revision.ApprenticeshipDetailsCorrect = ApprenticeshipDetailsCorrect;
-            revision.HowApprenticeshipDeliveredCorrect = HowApprenticeshipDeliveredCorrect;
-            revision.RolesAndResponsibilitiesConfirmations = RolesAndResponsibilitiesConfirmations;
+            if (DeliveryModelIsEquivalent()) revision.HowApprenticeshipDeliveredCorrect = HowApprenticeshipDeliveredCorrect;
```

Comment:
Could these two `if` branches be combined into one?

Reply:
yes. they can. Will do when I touch this code again